src/pages/SignUpPage/signUp.styles.css

```diff
@@ -0,0 +1,26 @@
+.formSignUp {
+    border: 1px solid blue;
+    border-radius: 10px;
+    width:60%;
+    padding: 3%;
+    margin: auto;
+    margin-top: 5%;
+    box-shadow: 5px 7px #456fa7;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+.text-field{
```

Comment:
this type of thing is much better to use `w-100` on the component itself, so future coders can just see quickly